src/main/java/org/betonquest/betonquest/compatibility/holographicdisplays/HologramLoop.java

```diff
@@ -53,17 +56,19 @@ public HologramLoop() {
                 if (!Bukkit.getPluginManager().isPluginEnabled("ProtocolLib")) {
                     LOG.warn(pack, "Holograms won't be able to hide from players without ProtocolLib plugin! "
                             + "Install it to use conditioned holograms.");
-                    runnable = null;
+                    //runnable = null;
```

Comment:
Is this supposed to be here 👀

Reply:
No. No, this should not be in there anymore... This variable does not even exist anymore. I'm fixing it right now.